app/mailers/reservation_mailer.rb

```diff
@@ -0,0 +1,11 @@
+class ReservationMailer < ApplicationMailer
+  default from: 'reservations@gmail.com'
+
+  def confirmed(reservation)
+    @reservation = reservation
+    @line_items = reservation.line_items.includes(:product)
+    attachments.inline['wallpaper'] = File.read('app/assets/images/img_01.jpg')
```

Comment:
why attaching a wallpaper?
what is inline?
what other options are there?

Reply:
Sir, attaching wallpaper just to try inline attachments in mails.
inline method calling on attachment turns it into an inline attachment.
Can't find other methods like inline that can be called on attachments.
We can use mime_type and encoding options for attachments.